# RNAseq tools
#### [Fastp](https://github.com/OpenGene/fastp): Preprocessing and quality control for raw FastQ data.
#### [FastQC](https://github.com/s-andrews/FastQC): Quality assessment of sequencing data after preprocessing.
#### [MultiQC](https://github.com/MultiQC/MultiQC): Aggregates and summarizes multiple QC reports.
#### [hisat2](https://github.com/DaehwanKimLab/hisat2): High-performance alignment of RNA-Seq reads to a reference genome.
#### [samtools](https://www.htslib.org/): Processing and manipulation of SAM/BAM files.
#### [featureCounts](https://rnnh.github.io/bioinfo-notebook/docs/featureCounts.html): Quantification of gene-level expression counts.

---

# DNAseq tools
###### DNAseq Tools

#### [FastQC](https://github.com/s-andrews/FastQC): Quality control of raw FastQ data to assess sequencing quality.
#### [MultiQC](https://github.com/MultiQC/MultiQC): Combines multiple QC reports into a single summary.
#### [BWA](https://github.com/lh3/bwa): Alignment of DNA sequencing reads to a reference genome.
#### [samtools](https://www.htslib.org/): Processing, sorting, and indexing of SAM/BAM files.
#### [Picard](https://broadinstitute.github.io/picard/): Marking duplicates and calculating alignment metrics.
#### [GATK](https://github.com/broadinstitute/gatk): Variant calling and filtering to generate VCF files.
#### [vcftools](https://vcftools.github.io/): Analysis and filtering of VCF files.

---

# DNA Methylation Analysis Tools

#### [FastQC](https://github.com/s-andrews/FastQC): Quality control for raw FastQ methylation data.
#### [Trim Galore](https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/): Trimming of adapters and low-quality bases for methylation sequencing.
#### [Bismark](https://github.com/FelixKrueger/Bismark): Alignment and methylation extraction from bisulfite-treated sequencing data.
#### [samtools](https://www.htslib.org/): Conversion, sorting, and indexing of SAM/BAM files from methylation alignments.
#### [picard](https://broadinstitute.github.io/picard/): Mark duplicates for bisulfite sequencing data.
#### [MethylKit](https://github.com/al2na/methylKit): Differential methylation analysis and visualization of methylation data.
#### [BS-Seeker2](https://github.com/BS-Seeker/BSseeker2): Bisulfite sequencing alignment and methylation calling.
#### [DeepTools](https://deeptools.readthedocs.io/en/develop/): Visualization of methylation and coverage data.
#### [methpipe](https://github.com/smithlabcode/methpipe): Comprehensive analysis pipeline for methylation data.


